### ISSUE TITLE
"Localization Energy Range" -- new property

### ISSUE THREAD

**Issue by shb46** [nasa-gcn/gcn-schema#204]
Add "localization_energy_range" to the "localization" schema for the range used by the localization calculation.

**Comment by Vidushi-GitHub** [comment]
@shb46 sorry I missed this, do you still request these changes?

**Comment by shb46** [comment]
@Vidushi-GitHub: These are not request, just suggestions for enhancements.

**Comment by Vidushi-GitHub** [comment]
Does GBM require this field?
We are happy to add it if some mission plans to use it.

**Comment by shb46** [comment]
@Vidushi-GitHub: Yes we use it. See: https://github.com/shb46/gcn-schema/blob/3812a3b32f353effb73dce64b6148fe1f1ef9ac9/gcn/notices/fermi/gbm/notice.schema.json#L93
